testing_template.html

```diff
@@ -27,7 +27,18 @@ <h1>Hello</h1>
 
 <p> In view.py you can see what the greeting variable looks like.</p> -->
 
+<!-- 
 
+template for IF/ELSE TAG
+{% if greeting ==1%}
```

Comment:
shouldn't this line following be indented?

Reply:
indentation added